nix/weaviate.nix

```diff
@@ -0,0 +1,102 @@
+{ pkgs, lib, name, config, ... }:
+let
+  inherit (lib) types;
+in
+{
+  options = {
+    enable = lib.mkEnableOption name;
+
+    package = lib.mkPackageOption pkgs "weaviate" { };
+
+    dataDir = lib.mkOption {
+      type = types.str;
+      default = "./data";
```

Comment:
Shouldn't this be `./data/${name}`?

<img width="379" alt="image" src="https://github.com/juspay/services-flake/assets/3998/ca78fbb7-8591-4f91-8c42-4e3fc7915f11">